code/modules/mob/living/silicon/ai/decentralized/management/ai_controlpanel.dm

```diff
@@ -197,13 +197,6 @@ GLOBAL_VAR_INIT(ai_control_code, random_nukecode(6))
 		data["current_ai_ref"] = REF(user)
 
 	data["can_log_out"] = !one_time_password_used
-
-	for(var/mob/living/silicon/ai/A in GLOB.ai_list)
```

Comment:
you uhh, can't just remove this, just remove the hijacking part

Reply:
whoops

Reply:
will be fixed next push